docs/examples/pydata.ipynb

```diff
@@ -24,7 +24,7 @@
     "import numpy as np\n",
     "import pandas as pd\n",
     "\n",
-    "rng = np.random.default_rng()\n",
+    "rng = np.random.default_rng(seed=15485863)\n",
```

Comment:
any reason to pick this seed ? why not a little easter egg like 42 ? 
yes this is a completely useless comment ;-)

Reply:
It's the one millionth prime. It's just a habit because for the old default RNG (mersenne twist) large primes were considered good choices for seeds. Here we're seeking stability not randomness so the choice of seed really doesn't matter, *and* the default RNG is no longer mersenne.

So, feel free to change if you want. A colleague of mine likes to use 8675309 as an Easter egg seed, in case you think 42 is too obvious

Reply:
So it is an easter egg indeed ;-)